drivers/SmartThings/zigbee-dimmer-remote/src/init.lua

```diff
@@ -15,6 +15,20 @@
 local capabilities = require "st.capabilities"
 local ZigbeeDriver = require "st.zigbee"
 local defaults = require "st.zigbee.defaults"
+local zcl_clusters = require "st.zigbee.zcl.clusters"
+
+local battery_attribute_configuration = {
+  cluster = zcl_clusters.PowerConfiguration.ID,
+  attribute = zcl_clusters.PowerConfiguration.attributes.BatteryPercentageRemaining.ID,
+  minimum_interval = 30,
+  maximum_interval = 14300, -- ~4hrs
+  data_type = zcl_clusters.PowerConfiguration.attributes.BatteryPercentageRemaining.base_type,
+  reportable_change = 1
+}
+
+local function device_init(driver, device)
+    device:add_configured_attribute(battery_attribute_configuration)
+end
```

Comment:
This change is because I noticed my device was cycling device health state since it didn't have any IASZone reporting to make it so it would report fast enough (like most battery powered devices do). The default battery reporting interval was not fast enough to keep devices online.

Note this will not fix existing devices, but any newly onboarded device will get the correct configuration. I tried to address devices in my testing, but I could not reliably get the sengled device to stay awake long enough to take the new configuration.